### DISCUSSION TITLE
When to read up to track 84?

### DISCUSSION THREAD

**Discussion by MoombaDS** [keirf/greaseweazle#483] [Q&A] [answered]
I have a lot of PC-98 disks with weird formats and copy protections that I'd like to archive as perfectly as possible in the hopes that someday support will improve enough that I can write them back as working disks.

My question is, if I want to get as accurate backups as possible for archival, should I be reading up to track 84 for every disk, just in case there's necessary data there? I struggle to know when I should be reading past track 80 and when it's unnecessary for raw dumps!

**Comment by keirf** [comment, marked as answer]
Copy protected disks may read past the usual 80 cylinders so why not read all 84 if your drive can manage it. I will say that on Amiga it was rare. I know of one title that used 83 cylinders and that got fixed soon after because not all drives could do 83 cylinders! 81 or 82 cylinders was a bit more common (but not very common at all).

**Reply by MoombaDS** [reply]
That makes sense, thanks!

In that case, would it be relatively fair to assume that if a disk drive pulled from an original machine isn't able to read up to 84 tracks, then it's not likely there would ever be data included on those tracks for that format in the first place? i.e. using an original drive and going for the max tracks it supports should always capture all relevant data (if both are in good shape, of course)?

**Reply by keirf** [reply]
I'd like to think so, but my Amiga example does disprove that in one instance. The title Stardust used 83 cylinders and would not work with all Amiga drives. Oops.
